src/propagate-job.ts

```diff
@@ -123,9 +147,7 @@ export async function run(): Promise<void> {
         async isWorkableCheck() {
           // Provide the function with a function to re-check whether the deposits can still be updated
           // after a batch of bundles fails to be included
-          const canUpdate = await job.canUpdateDeposits();
-          if (!canUpdate) console.log('Deposits are not updateable at this time. Restarting the script...');
-          return canUpdate;
+          return true;
```

Comment:
this needs to perform some sort of check so the function can know when to stop retrying sending the bundles. otherwise in an scenario where another relayer or user propagates the root, the script will keep retrying because it believes the function call would still succeed. 

some of the possible checks:
- fetching `aggregateRoot` and `lastPropagatedRoot` ensuring they are different
- doing a callStatic with the transaction and returning false if it fails in this check